contrib/analysis/merge.py

```diff
@@ -39,10 +39,10 @@ def main(args):
     thr_df = df[df["sample.measurement_type"] == "throughput"]
 
     # Covert dataframes to csv files
-    slat_df.to_csv(r"latency_slat.csv", sep=";", mode="a")
-    clat_df.to_csv(r"latency_clat.csv", sep=";", mode="a")
-    lat_df.to_csv(r"latency_lat.csv", sep=";", mode="a")
-    thr_df.to_csv(r"throughput_iops_sec.csv", sep=";", mode="a")
+    slat_df.to_csv(r"latency_slat.csv", sep=";", mode="w")
+    clat_df.to_csv(r"latency_clat.csv", sep=";", mode="w")
+    lat_df.to_csv(r"latency_lat.csv", sep=";", mode="w")
+    thr_df.to_csv(r"throughput_iops_sec.csv", sep=";", mode="w")
```

Comment:
I don't have a context but are you sure that these files only get written once and if they do get overwritten, we don't care about the first copy of it?

Reply:
They should only get written once - after all the processing is done these are the final output files. The error I was getting was when performing multiple executions of the script, the files were being appended to causing duplicates which was a problem. So having it be overwritten is desirable at least at this stage.